NuRadioMC/utilities/Veff.py

```diff
@@ -276,7 +276,10 @@ def get_Veff_Aeff_single(
     else:
         raise AttributeError(f"attributes do neither contain volume nor area")
 
-    Vrms = fin.attrs['Vrms']
+    try:
+        Vrms = fin.attrs['Vrms']
+    except:
+        Vrms = 1
```

Comment:
hm, why is `Vrms` not available? It should? Is `1` a good default value? I think you might be able to make this code a one-liner `Vrms = fin.attrs.get('Vrms', 1)` if hdf5 attributes are compatible with python dictionaries.